backend/hitas/migrations/0051_add_condition_of_sale.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.2.16 on 2023-01-18 18:47
```

Comment:
> Remove ownership uuid

Instead of adding it and removing it, can you instead delete the original commit that added it?

Reply:
I think I have 10+ commits on this branch between when I added the Onwership UUID and removing it, I would rather not go through the rebase hell of rewriting all those commits. I can do a squash at the end if we want a cleaner history.